### ISSUE TITLE
[question] support for ttf fonts?

### ISSUE THREAD

**Issue by japrogramer** [sapphire-arches/stgl#2]
Hello, since this terminal is opengl rendered. could support for ttf fonts be added?

**Comment by sapphire-arches** [comment]
The `font` name in `config.h` is a FontConfig query string and I just pass the result to Freetype2 so you should be able to point that at pretty much whatever font file you like.

**Comment by japrogramer** [comment]
Thank you, it looks like the st terminal source also has this setting but they have it configured differently. They don't use a separate variable for the font size.

static char font[] = "mononoki:pixelsize=16:antialias=false:autohint=false";

**Comment by sapphire-arches** [comment]
Yeah, because I knocked this together so quickly it's not really super clean. Setting font size using that option broke things and I haven't tracked down why that is just yet.
